scripts/asm-installer/install_asm

```diff
@@ -682,7 +684,11 @@ OPTIONS:
                                       Name of the ca in the GCP CAS service used to
                                       sign certificates in the format
                                       'projects/project_name/locations/ \
-                                       ca_region/certificateAuthorities/ca_name'
+                                      ca_region/certificateAuthorities/ca_name'
+  --revision       <CUSTOM_REVISION>  Custom revision label. Label needs to follow DNS
+                                      label formats (re: RFC 1123). Not supported if
+                                      control plane is managed. Recommended format:
+                                      asm-<major>.<minor>.<point>-revisionTag
```

Comment:
This is RFC 1123, compliant, but the revision also needs to be a valid Kubernetes label, which doesn't allow periods: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names

Reply:
corrected regex to not allow this.